docs/guides/admin/docs/configuration/database.md

```diff
@@ -68,10 +68,19 @@ create a database called `opencast` by executing:
 
 Then create a user `opencast` with a password and grant it all necessary rights:
 
-    GRANT SELECT,INSERT,UPDATE,DELETE,CREATE,DROP,INDEX,TRIGGER,CREATE TEMPORARY TABLES,REFERENCES ON opencast.*
+    GRANT SELECT,INSERT,UPDATE,DELETE ON opencast.*
```

Comment:
```suggestion
    GRANT SELECT,INSERT,UPDATE,DELETE,CREATE TEMPORARY TABLES ON opencast.*
```
`CREATE TEMPORARY TABLES` is a runtime requirement.

Reply:
This suggestion feature is cool and all, but where is the button to "amend last commit with suggestion" or something like that? x)